src/app/client/src/app/modules/user-onboarding/user-onboarding.module.ts

```diff
@@ -0,0 +1,27 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { OnboardingPopupComponent } from './components/onboarding-popup/onboarding-popup.component';
```

Comment:
@princegupta1131 
Add index for components folder and then import from index

Reply:
created index for components